app/controllers/profiles_controller.rb

```diff
@@ -38,11 +42,6 @@ def changed_email?
     user_params[:email].present?
   end
 
-  def set_mail_changes
-    flash[:notice] = I18n.t('flash.notice.profile_email_updated')
-    force_send_confirmation_email if @user.email.blank?
```

Comment:
@tauil @lucianosousa I was in doubt about the sign up/sign in behaviour.

as you guys can see, there's a spec indicating twitter sign up - without e-mail - is allowed, but simulating the sign up in my local env, I could not create an account through twitter: https://github.com/call4paperz/call4paperz/blob/master/spec/controllers/profiles_controller_spec.rb#L33 

I just changed as little as possible to make devise `3.5.10`work without break any test.

- why twitter sign up is not allowed?
- should users table have a not null clause for email field?